packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleTableColumnDateTextTests.cs

```diff
@@ -0,0 +1,238 @@
+
+        var expectedMarkup = @"custom-era=""short""";
+        Assert.Contains(expectedMarkup, table.Markup);
+    }
+
+    [Fact]
+    public void NimbleTableColumnDateText_WithCustomYearAttribute_HasTableMarkup()
+    {
+        var table = RenderWithPropertySet(x => x.CustomYear, YearFormat.TwoDigit);
+
+        var expectedMarkup = @"custom-year=""2-digit""";
+        Assert.Contains(expectedMarkup, table.Markup);
+    }
+
+    [Fact]
+    public void NimbleTableColumnDateText_WithCustomMonthAttribute_HasTableMarkup()
+    {
+        var table = RenderWithPropertySet(x => x.CustomMonth, MonthFormat.Long);
+
+        var expectedMarkup = @"custom-month=""long""";
+        Assert.Contains(expectedMarkup, table.Markup);
+    }
+
+    [Fact]
+    public void NimbleTableColumnDateText_WithCustomDayAttribute_HasTableMarkup()
+    {
+        var table = RenderWithPropertySet(x => x.CustomDay, DayFormat.Numeric);
+
+        var expectedMarkup = @"custom-day=""numeric""";
+        Assert.Contains(expectedMarkup, table.Markup);
+    }
+
+    [Fact]
+    public void NimbleTableColumnDateText_WithCustomHourAttribute_HasTableMarkup()
+    {
+        var table = RenderWithPropertySet(x => x.CustomHour, HourFormat.Numeric);
+
+        var expectedMarkup = @"custom-hour=""numeric""";
+        Assert.Contains(expectedMarkup, table.Markup);
+    }
+
+    [Fact]
+    public void NimbleTableColumnDateText_WithCustomMinuteAttribute_HasTableMarkup()
+    {
+        var table = RenderWithPropertySet(x => x.CustomMinute, MinuteFormat.TwoDigit);
+
+        var expectedMarkup = @"custom-minute=""2-digit""";
+        Assert.Contains(expectedMarkup, table.Markup);
+    }
+
+    [Fact]
+    public void NimbleTableColumnDateText_WithCustomSecondAttribute_HasTableMarkup()
+    {
+        var table = RenderWithPropertySet(x => x.CustomSecond, SecondFormat.Numeric);
+
+        var expectedMarkup = @"custom-second=""numeric""";
+        Assert.Contains(expectedMarkup, table.Markup);
+    }
+
+    [Fact]
+    public void NimbleTableColumnDateText_WithCustomTimeZoneNameAttribute_HasTableMarkup()
+    {
+        var table = RenderWithPropertySet(x => x.CustomTimeZoneName, TimeZoneNameFormat.ShortGeneric);
+
+        var expectedMarkup = @"custom-time-zone-name=""shortGeneric""";
+        Assert.Contains(expectedMarkup, table.Markup);
+    }
+
+    [Fact]
+    public void NimbleTableColumnDateText_WithCustomFormatMatcherAttribute_HasTableMarkup()
+    {
+        var table = RenderWithPropertySet(x => x.CustomFormatMatcher, FormatMatcherAlgorithm.Basic);
+
+        var expectedMarkup = @"custom-format-matcher=""basic""";
+        Assert.Contains(expectedMarkup, table.Markup);
+    }
+
+    [Fact]
+    public void NimbleTableColumnDateText_WithCustomHour12Attribute_HasTableMarkup()
+    {
+        var table = RenderWithPropertySet(x => x.CustomHour12, false);
+
+        var expectedMarkup = @"custom-hour12=""false""";
```

Comment:
This test is currently failing because the `custom-hour12` attribute is behaving like a standard boolean. There is a change in the Angular support PR that fixes this (in nimble-components). When that PR goes in, I'll merge and the test should pass.